### PR TITLE
type changes in databaseDetails page

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Change type of databaseDetails

#
### Type of change :

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @shahsank3t, @darth-coder00, @Sachin-chaurasiya

### CODE REVIEW

File: catalog-rest-service/src/main/resources/ui/src/pages/database-details/index.tsx [hunk 1]
```diff
@@ -328,12 +330,16 @@ const DatabaseDetails: FunctionComponent = () => {
                       key={index}>
                       <td className="tableBody-cell">
                         <Link
-                          to={getDatasetDetailsPath(table.fullyQualifiedName)}>
+                          to={
+                            table.fullyQualifiedName
+                              ? getDatasetDetailsPath(table.fullyQualifiedName)
+                              : '#'
```
Comment (darth-coder00) [inline catalog-rest-service/src/main/resources/ui/src/pages/database-details/index.tsx:336]: Why `#` here?

**Review by darth-coder00 — COMMENTED** [review]
Just one comment, otherwise nicely done 👍

**Review by Sachin-chaurasiya — APPROVED** [review]
LGTM

File: catalog-rest-service/src/main/resources/ui/src/pages/database-details/index.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply catalog-rest-service/src/main/resources/ui/src/pages/database-details/index.tsx:336]: FQN is coming optional from types

**Pull request merged**